universal/router/React/index.js

```diff
@@ -0,0 +1,32 @@
+/* @flow */
+import React from 'react'
+
+import LayoutSidebar from 'layouts/LayoutSidebar'
+import Markdown from 'components/Markdown'
+import Counter from 'modules/Counter'
+
+import type { Route } from '../types'
+import match from '../matches'
```

Comment:
Maybe pull it out before you use it then. I also _feel_ that it reads poorly in the JSX. I would like it if we could just adopt something like [Airbnb's style guide](https://github.com/airbnb/javascript/tree/master/react#tags) so that our styling is less about feelings than documentation, except for maybe a few deviations that are documented in our ReadMe